dev/ledger.md

```diff
@@ -132,18 +132,63 @@ The block header contains the following components:
    protocol version that supported the transaction counter.  The counter is
    stored in msgpack field `tc`.
 
- - The block's _expired participation accounts_, which contains an optional slice of
-   public keys of accounts. These accounts are expected to have their participation
-   key expire by the end of the round (or was expired before the current round). The
-   msgpack representation of the components are described in detail below.
-   The slice is stored in msgpack key `partupdrmv`.
-
-The block's _expired participation accounts_ slice is valid as long as the participation
-keys of all the accounts in the slice are expired by the end of the round or were
-expired before, the accounts themselves would have been online at the end of the
-round if they were not included in the slice, and the number of elements in the slice is
-less or equal to 32. A block proposer may not include all such
-accounts in the slice and may even omit the slice completely.
+ - The block's _proposer_, which is the address of the account that
+   proposed the block. The proposer is stored in msgpack field `prp`.
+
+ - The block's _fees collected_ is the sum of all fees paid by transactions in
+   the block and is stored in msgpack field `fc`.
+
+ - The potential _bonus incentive_ is the amount, in MicroAlgos, that
+   may be paid to the proposer of this block beyond the amount
+   available from fees. It is stored in msgpack field `bi`.  It may be
+   set during a consensus upgrade, or else it must be equal to the
+   value from the previous block in most rounds, or be 99% of the
+   previous value (rounded down) if the round of this block is 0
+   modulo 1,000,000.
+
+ - The _proposer payout_ is the actual amount that is moved from the
+   $I_f$ to the proposer, and is stored in msgpack field `pp`. If the
+   proposer is not eligible, as described below, the _proposer payout_
+   must be 0. If the proposer is eligible, the proposer payout is
+   valid if it is less than or equal to the the current potential
+   payout, which is computed as the sum of the _bonus incentive_ and
+   half of the _fees collected_.
```

Comment:
@jannotti I think here the spec should specify what happens if the `FeeSink` does not have enough ALGO to cover the payout.

Reply:
Addressed in #107